src/static/riot/profile/profile.tag

```diff
@@ -307,6 +309,10 @@
                 .done(function (data) {
                     self.user = data
                     self.update()
+                    _.delay(() => {
+                        self.loading = false
+                        self.update()
+                    }, 500)
```

Comment:
maybe have a comment about why the long delay, like on v2?

```
// Not strictly necessary, but makes the loader show up long enough to be recognized as such,
// rather than a weird flicker
```